### PR TITLE
Bug 1755306 - Revert the unintended vxlan api backport

### PR DESCRIPTION
**Pull request by abhat**
Revert "Merge pull request #90 from JacobTanenbaum/release-3.10_vxlan_backport"

This reverts commit 7eef55a5dc24306c455777b9897ddc3cbeaf8b59, reversing
changes made to 6a90ae6466cec806da11feb3b15708ca33c1c54b.

### CODE REVIEW

**Comment by knobunc** [comment]
@deads2k can you please approve this?  The change went in because the bot jumped the gun... and we need to unwind it.

**Comment by deads2k** [comment]
I have a few questions before we merge this.

1. did we ship a level with the "bad" vxlan value?
2. using golang 1.12 to build openshift 3.10 seems weird.  When did 3.10 move to use 1.12?
3. Does anything bad happen if we leave this value in? Missing validation or something?

**Comment by abhat** [comment]
I think it's not so much a bad value. The VxLanPort wasn't supposed to be configurable in 3.10. The PR got accidentally merged on a /lgtm label when it didn't have a /approve label on it.
The go 1.12 version bump was needed to get the builds to pass. Not sure if the rest of 3.10 is built with 1.12 or not. Without bumping the go version we were hitting this build failure. https://travis-ci.com/openshift/api/builds/133453543#L258.

**Comment by abhat** [comment]
Additional details re: the build failure with go1.8: https://github.com/golang/go/issues/28291. TL;DR this is an issue with golint which may go away with vendoring a newer golint or bumping the go version itself. @deads2k Do you see a downside to using a later version of the toolchain?

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.8
```
Comment (squeed) [inline .travis.yml]: yeah, you can't do this. You'll need to find a 1.8 version to run gofmt :-) (or do it manually)

**Comment by deads2k** [comment]
> Casey Callendrello 10:14 AM
> @deads2k it seems that openshift/api was never bumped to pick up the change
> so I believe it's safe to revert
> correction: openshift/origin v3.10

/approve

**Pull request merged**